src/pages/home/ReportScreen.js

```diff
@@ -291,8 +292,8 @@ export default compose(withNetwork(), withOnyx({
     betas: {
         key: ONYXKEYS.BETAS,
     },
-    isLoadingInitialReportActions: {
-        key: ({route}) => `${ONYXKEYS.COLLECTION.IS_LOADING_INITIAL_REPORT_ACTIONS}${getReportID(route)}`,
+    isLoadingReportActions: {
+        key: ({route}) => `${ONYXKEYS.COLLECTION.IS_LOADING_REPORT_ACTIONS}${getReportID(route)}`,
         initWithStoredValues: false,
```

Comment:
I guess one downside of storing this on the `report` will be that we cannot use `initWithStoredValues: false` 🤔 

Though, I'm not sure if it's necessary to do this.